lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -1175,6 +1134,21 @@ def get_workflow_menu(
             payload=payload,
         )
 
+    @router.get(
+        "/api/workflows/{encoded_workflow_id}",
+        summary="Displays information needed to run a workflow.",
+        name="show_workflow",
+    )
+    def show_workflow(
+        self,
+        encoded_workflow_id: StoredWorkflowIDPathParam,
+        instance: InstanceQueryParam = False,
+        legacy: LegacyQueryParam = False,
+        version: VersionQueryParam = None,
+        trans: ProvidesHistoryContext = DependsOnTrans,
+    ):
+        return self.service.show_workflow(trans, encoded_workflow_id, instance, legacy, version)
+
```

Comment:
Renaming `encoded_workflow_id` to `workflow_id` should fix the API linting error. But don't ask me why... :laughing: 

Anyway, it is more consistent with the rest of the endpoints.

```suggestion
    @router.get(
        "/api/workflows/{workflow_id}",
        summary="Displays information needed to run a workflow.",
        name="show_workflow",
    )
    def show_workflow(
        self,
        workflow_id: StoredWorkflowIDPathParam,
        instance: InstanceQueryParam = False,
        legacy: LegacyQueryParam = False,
        version: VersionQueryParam = None,
        trans: ProvidesHistoryContext = DependsOnTrans,
    ):
        return self.service.show_workflow(trans, workflow_id, instance, legacy, version)

```